spec/mailers/previews/applicant_mailer_preview.rb

```diff
@@ -7,4 +7,9 @@ def applicant_accept
     ApplicantMailer.applicant_accept(applicant)
   end
 
+  def application_request
+    applicant = Applicant.first
```

Comment:
Create an applicant then use them for this test. alternatively use the first applicant in your fixtures and make sure they are accepted